packages/web-app-draw-io/src/App.vue

```diff
@@ -212,11 +212,12 @@ export default defineComponent({
         previousEntityTag: this.currentETag
       })
         .then((resp) => {
-          this.currentETag = resp.ETag
+          // FIXME: is this correct? The type says ... yes
+          this.currentETag = resp.etag
```

Comment:
yup, the change is correct - we're dealing with resources here (at least since we introduced and ported this to `web-client`) and it's always `etag`. No matter if public or private spaces..